src/main/java/htsjdk/samtools/SAMUtils.java

```diff
@@ -1254,7 +1254,7 @@ public static boolean isValidUnsignedIntegerAttribute(long value) {
      * Checks if reference sequence is compatible with BAI indexing format.
      * @param sequence reference sequence.
      */
-    public static boolean isReferenceSequenceCompatibleWithBAI(final SAMSequenceRecord sequence) {
```

Comment:
Another change in public API....perhaps keep the old, misleading name around (deprecated), with a comment?

Reply:
Done.